### DISCUSSION TITLE
Soroban Prologue 3D Submission

### DISCUSSION THREAD

**Discussion by Polscche** [stellar/sorobanathon#174] [General]
3D Entry for the soroban Quest by PausePlayed
https://litemint.com/items/GD2O3W4QKTEXWYEBEW5AYO7ZWI3JXMYAEWHVDKNFUHGI6WKLIB65UN5Z/Prologue
![5](https://user-images.githubusercontent.com/62219907/208187254-1d1d207f-190f-4a5c-95fe-8cda8a64d71d.png)

![4](https://user-images.githubusercontent.com/62219907/208187289-ddbd6eb5-d7c4-4218-b5fb-e6509b648214.png)

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
